### PR TITLE
Devops 1238 add es tls to templating

### PR DESCRIPTION
**Pull request by bob-bins**

### CODE REVIEW

File: sysdigcloud/config.yaml [hunk 1]
```diff
@@ -103,14 +103,14 @@ data:
...
+  elasticsearch.hostname: sysdigcloud-elasticsearch
   # Optional: Elasticsearch JVM options. For heavy load environments this will override the defaults (e.g. -Xms6G -Xmx6G)
   elasticsearch.jvm.options: ""
-  # Optional: Elasticsearch JVM options. Suggested value "-Xms6G -Xmx6G"
-  elasticsearch.jvm.options: ""
-  #Username required for the Search Guard authentication
+  # Username required for the Search Guard authentication
   elasticsearch.searchguard.enabled: "false"
   elasticsearch.user: "admin"
+  # This needs to be false when using Search Guard, otherwise set to true
```
Comment (mbreitung) [inline sysdigcloud/config.yaml]: Could this comment be removed? We could force `verifySSL=false` when we find that `  elasticsearch.searchguard.enabled: "true"`?

Comment (mbreitung) [reply sysdigcloud/config.yaml]: I think that the verifySSL needs to remain for the scenario where the customer is using a self hosted ES with a real cert

Comment (bob-bins) [reply sysdigcloud/config.yaml]: What do you think about changing the value to `false` by default and updating the comment to say something like `You may change this to true if you are running your own secure Elasticsearch cluster and desire hostname verification`.

I don't love the idea of having an exposed configurable parameter be possibly ignored based on the value of other parameters. It's a place for future confusion

Comment (bob-bins) [reply sysdigcloud/config.yaml]: ended up commenting the property out, saying it should only be used if the user has their own secure ES cluster. Backend has code to set the value appropriately when Searchguard is enabled

File: datastores/as_kubernetes_pods/manifests/elasticsearch/README.md [hunk 2]
```diff
@@ -1,21 +1,113 @@
...
+
+        NAMESPACE=<replace-this>
+        kubectl -n $NAMESPACE create secret generic ca-certs --from-file=<path to root-ca.pem>
+
+3. Uncomment `elasticsearch.verifySSL` in `sysdig/config.yaml` and set the value appropriately.
+
+4. Follow steps 4, 7, and 8 from [Enabling authentication on an existing cluster with standard Sysdig Elasticsearch setup](#enabling-authentication-on-an-existing-cluster-with-standard-sysdig-elasticsearch-setup).
+
+# TODO: review all content below this to ensure correctness
 # Search Guard Configuration Files
```
Comment (bob-bins) [inline datastores/as_kubernetes_pods/manifests/elasticsearch/README.md]: If we want to initially keep things simple, I'm questioning whether this should be an option for the customer. If so, I will need to enhance this section

**Comment by bob-bins** [comment]
@mbreitung should I create another branch that does not have `configurator` changes and submit a PR to `master`?

**Comment by janoe** [comment]
Is this PR still valid? I see that the new code is reverting the newer changes

**Comment by bob-bins** [comment]
replaced by https://github.com/draios/sysdigcloud-kubernetes/pull/216

**Comment by bob-bins** [comment]
sorry @janoe, I missed your comment!

**Pull request closed without merging**